src/main/java/domain/Lotto.java

```diff
@@ -1,104 +1,84 @@
 package domain;
 
 import dto.LottoMatchResult;
-import util.NumberPicker;
-
-import java.util.ArrayList;
-import java.util.Collections;
 import java.util.HashSet;
 import java.util.List;
+import util.NumberPicker;
 
 public class Lotto {
-    
+
     private static final int START_NUMBER = 1;
     private static final int END_NUMBER = 45;
     private static final int NUMBER_COUNT = 6;
     private static final int PRICE = 1000;
-    
+
     private final List<Integer> numbers;
-    
+
     public Lotto(List<Integer> numbers) {
         validateNumberCount(numbers);
         validateNumberNotDuplicated(numbers);
         this.numbers = numbers;
     }
-    
+
     private void validateNumberCount(List<Integer> numbers) {
         if (numbers.size() != NUMBER_COUNT) {
             throw new IllegalArgumentException("로또 번호는 6개가 되어야 합니다.");
         }
     }
-    
+
     private void validateNumberNotDuplicated(List<Integer> numbers) {
         if (numbers.size() != new HashSet<>(numbers).size()) {
             throw new IllegalArgumentException("로또 번호는 중복되면 안됩니다.");
         }
     }
-    
-    public static List<Lotto> purchase(int money, NumberPicker numberPicker) {
-        validateMinPrice(money);
-        
-        List<Lotto> lottos = new ArrayList<>();
-        int lottoCount = money / PRICE;
-        for (int i = 0; i < lottoCount; i++) {
-            List<Integer> numbers = numberPicker.pickUnique(START_NUMBER, END_NUMBER, NUMBER_COUNT);
-            Lotto lotto = new Lotto(numbers);
-            lottos.add(lotto);
-        }
-        return Collections.unmodifiableList(lottos);
+
+    public static int getPurchaseableCount(int money) {
```

Comment:
```suggestion
    public static int getPurchasableCount(int money) {
```



<img width="433" alt="스크린샷 2025-02-17 오후 9 21 02" src="https://github.com/user-attachments/assets/65d168f2-257c-42ec-b509-54fc31553a65" />

<img width="497" alt="스크린샷 2025-02-17 오후 9 22 00" src="https://github.com/user-attachments/assets/516e97eb-7ffc-483a-b6fd-c11501d572ad" />


인텔리제이에서 밑줄, 형광펜표시 등이 뜨는 것을 잘 확인해주세요 
친절하게 정확한 스펠링 추천까지 해준답니다🙂

Reply:
엇, 왠지 모르게 해당 기능이 꺼져있었네요! 감사합니다!!